### ISSUE TITLE
Did you go by Muhznit in 2007?

### ISSUE THREAD

**Issue by throwaway01** [Muhznit/hello-world#1]
This is definitely not what GitHub is for, but I'm incredibly curious and this seems like a decent shot. If so, I may have known you (online) ten years ago. Did you make "Explosions in MS Paint"?

**Comment by Muhznit** [comment]
Yeah, I made that a loooong while back. "Explosions in Mspaint". Far enough
in the past that I was known as "Gamedragon89" instead and used AOL. I've
used this name for a good 10 years or so by now. I can only guess at who
you are though. I'm guessing you were part of the BannedStory/MapleSim
forums? (outside of high school, most people knew me from there)
